__test__/pet.test.js

```diff
@@ -0,0 +1,96 @@
+const Pet = require('../src/pet');
+
+describe('constructor', () => {
+    it('returns an object', () => {
+      expect(new Pet('Ziggy')).toBeInstanceOf(Object);
+    });
+    it('sets the name property', () => {
+      const pet = new Pet('Ziggy');
+      expect(pet.name).toEqual('Ziggy')
+    });
+    it('has a initial age of 0', () => {
+      const pet = new Pet('Ziggy');
+      expect(pet.age).toEqual(0);
+    });
+    it('has initial hunger of 0', () => {
+      const pet = new Pet('Ziggy')
+      expect(pet.hunger).toEqual(0);
+    });
+    it('has initial fitness of 10', () => {
+      const pet = new Pet('Ziggy')
+      expect(pet.fitness).toEqual(10);
+    });
+  });
+  
+  describe('growUp', () => {
+    it('increments the age by 1', () => {
+      const pet = new Pet('Ziggy');
+      pet.growUp();
+      expect(pet.age).toEqual(1);
+    });
+    it('increments hunger by 5', () => {
+      const pet = new Pet('Ziggy');
+      pet.growUp();
+      expect(pet.hunger).toEqual(5);
+    });
+    it('decrease fitness by 3', () => {
+      const pet = new Pet('Ziggy');
+      pet.growUp();
+      expect(pet.fitness).toEqual(7);
+    });
+
+    describe('walk', () => {
+      it('increases fitness by 4', () => {
+        const pet = new Pet('Ziggy');
+        pet.fitness = 4;
+        pet.walk();
+        expect(pet.fitness).toEqual(8)
+      });
+     it('increases fitness to a maximum of 10', () => {
+      const pet = new Pet('Ziggy');
+      pet.fitness = 8;
+      pet.walk();
+      expect(pet.fitness).toEqual(10);
+     });
+    });
```

Comment:
Indentation issues as above
```suggestion
describe('walk', () => {
    it('increases fitness by 4', () => {
        const pet = new Pet('Ziggy');
        pet.fitness = 4;
        pet.walk();
        expect(pet.fitness).toEqual(8)
    });
    it('increases fitness to a maximum of 10', () => {
        const pet = new Pet('Ziggy');
        pet.fitness = 8;
        pet.walk();
        expect(pet.fitness).toEqual(10);
    });
});
```